avocado/utils/astring.py

```diff
@@ -29,8 +29,8 @@
 import re
 import string
 
-from six import string_types
-
+from six import string_types, PY3
+from six.moves import zip, xrange
```

Comment:
the comment from the other PR applies here. Let's give precedence to the Py3 syntax (range) over Py2's (xrange).

Reply:
True, forgot about it.

Reply:
Well the question is whether we want to keep using `py2to3`. If so than you can simply use `xrange` and it'll get converted to `range` during deployment, while it'll stay fast in python2 execution.

Reply:
On my Python 3 development branch, I've removed `2to3` support.  The goal being that the same code base, without the build step, can be used on both Python 2 and 3.